### PR TITLE
Update wording for privacy notice

### PR DESCRIPTION
**Pull request by brucebolt**
This reflects the text in https://docs.google.com/document/d/1Fa_O4666tGeRHskbcOwvpgrjulI3z7lHbUxlsJsG2x4/edit.

Trello card: https://trello.com/c/TFuoFNDS

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2,10 +2,58 @@ en:
...
+      <p class="govuk-body">The contact details for the data controller’s Data Protection Officer are: Stephen Jones, Data Protection Officer, Cabinet Office, 70 Whitehall, London, SW1A 2AS, or <a href="mailto:dpo@cabinetoffice.gov.uk">dpo@cabinetoffice.gov.uk.</a></p>
+      <p class="govuk-body">The Data Protection Officer provides independent advice and monitoring of Cabinet Office’s use of personal information.</p>
+      <p class="govuk-body">If you consider that your personal information has been misused or mishandled, you can make a complaint to the Information Commissioner, who is an independent regulator. The Information Commissioner can be contacted at:  Information Commissioner's Office, Wycliffe House, Water Lane, Wilmslow, Cheshire, SK9 5AF, or 0303 123 1113, or <a href="mailto:casework@ico.org.uk">casework@ico.org.uk</a>. Any complaint to the Information Commissioner won’t affect your right to seek redress through the courts.</p>
```
Comment (alex-ju) [inline config/locales/en.yml]:
```suggestion
      <p class="govuk-body">If you consider that your personal information has been misused or mishandled, you can make a complaint to the Information Commissioner, who is an independent regulator. The Information Commissioner can be contacted at:  Information Commissioner's Office, Wycliffe House, Water Lane, Wilmslow, Cheshire, SK9 5AF, or 0303 123 1113, or <a href="mailto:casework@ico.org.uk" class="govuk-link">casework@ico.org.uk</a>. Any complaint to the Information Commissioner won’t affect your right to seek redress through the courts.</p>
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -2,10 +2,58 @@ en:
...
+        <li>that any incomplete personal information is updated - you can include the missing information in your request</li>
+        <li>that your personal information is erased if there is no longer a justification for it to be processed</li>
+        <li>that the processing of your personal information is restricted in certain circumstances - for example, where accuracy is contested</li>
+      </ul>
+      <p class="govuk-body">If you want to make any of these requests, contact us using the details below.</p>
+
+      <h2 class="govuk-heading-m">Questions and complaints</h2>
+      <p class="govuk-body">The contact details for the data controller are: Cabinet Office (Government Digital Service), The White Chapel Building, 10 Whitechapel High Street, London, E1 8QS, or <a href="mailto:gds-privacy-office@digital.cabinet-office.gov.uk">gds-privacy-office@digital.cabinet-office.gov.uk</a></p>
+      <p class="govuk-body">The contact details for the data controller’s Data Protection Officer are: Stephen Jones, Data Protection Officer, Cabinet Office, 70 Whitehall, London, SW1A 2AS, or <a href="mailto:dpo@cabinetoffice.gov.uk">dpo@cabinetoffice.gov.uk.</a></p>
```
Comment (alex-ju) [inline config/locales/en.yml]:
```suggestion
      <p class="govuk-body">The contact details for the data controller’s Data Protection Officer are: Stephen Jones, Data Protection Officer, Cabinet Office, 70 Whitehall, London, SW1A 2AS, or <a href="mailto:dpo@cabinetoffice.gov.uk" class="govuk-link">dpo@cabinetoffice.gov.uk.</a></p>
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -2,10 +2,58 @@ en:
...
+        <li>information about how your personal information is processed, and to request a copy of that personal data</li>
+        <li>that any inaccuracies in your personal information is rectified without delay</li>
+        <li>that any incomplete personal information is updated - you can include the missing information in your request</li>
+        <li>that your personal information is erased if there is no longer a justification for it to be processed</li>
+        <li>that the processing of your personal information is restricted in certain circumstances - for example, where accuracy is contested</li>
+      </ul>
+      <p class="govuk-body">If you want to make any of these requests, contact us using the details below.</p>
+
+      <h2 class="govuk-heading-m">Questions and complaints</h2>
+      <p class="govuk-body">The contact details for the data controller are: Cabinet Office (Government Digital Service), The White Chapel Building, 10 Whitechapel High Street, London, E1 8QS, or <a href="mailto:gds-privacy-office@digital.cabinet-office.gov.uk">gds-privacy-office@digital.cabinet-office.gov.uk</a></p>
```
Comment (alex-ju) [inline config/locales/en.yml]:
```suggestion
      <p class="govuk-body">The contact details for the data controller are: Cabinet Office (Government Digital Service), The White Chapel Building, 10 Whitechapel High Street, London, E1 8QS, or <a href="mailto:gds-privacy-office@digital.cabinet-office.gov.uk" class="govuk-link">gds-privacy-office@digital.cabinet-office.gov.uk</a></p>
```

**Review by alex-ju — APPROVED** [review]
Left a few amends to cope with link styling. You can commit and squash at merge.

**Review by huwd — APPROVED** [review]

**Comment by huwd** [comment]
I read through the content and compared to the doc, looks great!

**Pull request merged**